daemon/graphdriver/quota/projectquota_test.go

```diff
@@ -0,0 +1,87 @@
+// +build linux
+
+package quota
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// 10MB
+const testQuotaSize = 10 * 1024 * 1024
+
+func TestQuota(t *testing.T) {
+	var err error
+	homeDir, err := ioutil.TempDir("", "docker-copy-check")
+	require.NoError(t, err)
+	defer os.RemoveAll(homeDir)
+
+	backingFsDev, err := makeBackingFsDev(homeDir)
+	require.NoError(t, err)
+
+	hasSupport, err := hasQuotaSupport(backingFsDev)
+	require.NoError(t, err)
+
+	if !hasSupport {
```

Comment:
Could we just make a small block device with quota support and one without?

Reply:
Done.

Reply:
Although, it's relying on mount options, not quota support (or not)

Reply:
That makes sense, what you are doing in the other test makes sense. For this test should we not do something similar? For example running the test locally `homeDir, err := ioutil.TempDir("", "docker-copy-check")` will always run the tests in `/tmp` without quota support. Maybe just make the device like the other test, mount it, and use that as root for these tests?